src/Symfony/Component/Security/Http/Tests/Firewall/SwitchUserListenerTest.php

```diff
@@ -406,4 +406,34 @@ public function testSwitchUserStateless()
         $this->assertInstanceOf(UsernamePasswordToken::class, $this->tokenStorage->getToken());
         $this->assertFalse($this->event->hasResponse());
     }
+
+    public function testSwitchUserRefreshesOriginalToken()
+    {
+        $originalUser = $this->getMockBuilder(\Symfony\Component\Security\Core\User\UserInterface::class)->getMock();
```

Comment:
Please use createMock and no FQCN, thanks

Reply:
I matched the style to the rest of the tests in the file, is it ok to use a different style from the file?

Reply:
This is the syntax applied since 5.1.

Reply:
Oh, I see this change was made to 4.4 the day after I opened the pull request, I hadn't noticed that. I've rebased on top of the latest 4.4 and fixed the issue.

Reply:
Correction, _now_ I've fixed the issue. Silly me I rebased and fixed it but forgot to commit the changes I made...